### ISSUE TITLE
lightdm-settings & profile 17.1 woes? 

### ISSUE THREAD

**Issue by SleepyMario** [jorgicio/jorgicio-gentoo-overlay#186]
Hello, when trying to run lightdm-settings-1.2.8, I'm running into the issue below:

-------
Traceback (most recent call last):
  File "/usr/bin/lightdm-settings", line 11, in <module>
    subprocess.Popen("/usr/lib/lightdm-settings/lightdm-settings")
  File "/usr/lib64/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/usr/lib/lightdm-settings/lightdm-settings': '/usr/lib/lightdm-settings/lightdm-settings'
-----

I suspect this has to do with the recent profile change to 17.1? Making a symlink from /usr/lib64/lightdm-settiings to /usr/lib does work here as a solution.

Then another problem arises which I'm not sure it's related, so I'll mention it here; feel free to change it to whenever you want or I'll open a new issue if that's better. 

Basically when I change the wallpaper from within lightdm-settings it will not change it unless I change the name of the former wallpaper (as if it's being overwritten when lightdm is loaded upon boot). 

Thanks for your help!
